plugins/woocommerce/client/blocks/assets/js/atomic/blocks/product-elements/sale-badge/index.tsx

```diff
@@ -20,7 +19,7 @@ const blockConfig = {
 		/>
 	),
 	edit,
-	save,
+	save: () => null,
```

Comment:
If someone had a block inserted in Single Product block, this will result in validation error at first check. But I guess it's fine and worth the change! I don't think we should go through `migrate` from deprecation API for this edge case but mentioning it just in case 🤔

Reply:
Good point! I implemented the migration with 941997f5abe03ad045c5ed5f313b5d29afa8d84b

Reply:
Amazing job with investigating the deprecation API!!! <3